src/test/java/com/xmslyz/luhn/logic/LuhnChecksumComputingTest.java

```diff
@@ -0,0 +1,98 @@
+package com.xmslyz.luhn.logic;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+class LuhnChecksumComputingTest {
+
+    @SuppressWarnings("all")
+    private Computation computation = new LuhnChecksumComputing();
```

Comment:
Lepiej naprawić, niż pisać `SuppressWarnings("all")` - przez to wyłączasz wszystkie `warn`y, które moga pojawić sie w przyszłości. Najlepiej unikać `SuppressWarnings` jeśli tylko można, bo mogą ukryć coś co jest poważniejszym problemem. Ba, lepiej zostawić warna - niech bije po oczach przy budowaniu. warn mówi, że coś można poprawić, ale nie psuje to działania programu (najpewniej - patrz [code smell](https://pl.wikipedia.org/wiki/Zapachy_kodu))  
Tutaj naprawa jest stosunkowo prosta:
```suggestion
    private final Computation computation = new LuhnChecksumComputing();
```
Dlaczego dostałeś warn? Analiza kodu zauważyła, że nigdzie nie zmieniasz wartości `computation`, więc można oznaczyć je jako stałą. Używając `final` nigdy nie będziesz mógł przypisać nic innego do `computation`.  To bardzo ważna informacja, bo znacząco ułatwia jego obsługę i zrozumienie działania kodu. Im więcej stałych i mniej zmiennych, tym łatwiej ogarnąć co się w algorytmie dzieje.